test/SaveHandler/DbTableGateway/PgsqlAdapterTest.php

```diff
@@ -4,13 +4,13 @@
 
 use Laminas\Db\Adapter\Adapter;
 use Laminas\Session\SaveHandler\DbTableGateway;
-use LaminasTest\Session\SaveHandler\DbTableGatewayTest;
+use LaminasTest\Session\SaveHandler\AbstractDbTableGatewayTest;
 
 use function extension_loaded;
 use function getenv;
 use function sprintf;
 
-class PgsqlAdapterTest extends DbTableGatewayTest
+class PgsqlAdapterTest extends AbstractDbTableGatewayTest
```

Comment:
Check line 29 of this file - it's checking for the mysqli extension, and not pgsql, when determining whether or not it can run.